spec/factories/686c/form_686c_674.rb

```diff
@@ -132,7 +132,8 @@
               'suffix' => 'II'
             },
             'ssn' => '370947141',
-            'birth_date' => '2001-03-03'
+            'birth_date' => '2001-03-03',
+            'dependent_income' => true
```

Comment:
Does this need to be a Y/N value?

Reply:
This value comes in as true/false from the front end, and then we have to convert it to Y/N before sending it to our back end partner.  So for the spec payload data I have it coming in as true to mock how it would be coming in from the FE.  In the spec test, it will convert the value to Y.

Reply:
Ah, I see!